### PR TITLE
Update species stats data to 104

### PR DESCRIPTION
**Pull request by msmanoj**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1157

## Description
Updated the sample data file for species stats

## Deployment URL
http://update-species-stats.review.ensembl.org

## Views affected
Specie stats

### Other effects
- [ ] I have checked any requirements for Google Analytics
- [ ] I have created any required tests
- [ ] The PR has as its final commit a WASM/JS update commit if any Rust files were updated.

### CODE REVIEW

File: src/ensembl/src/content/app/species/sample-data.ts [hunk 1]
```diff
@@ -40,66 +40,66 @@ export const sampleData: RawSpeciesStats = {
...
-      structural_variants: 32132047,
-      short_variants_with_phenotype_assertions: 13336864,
-      short_variants_with_publications: 303288,
+      short_variants: 713749788,
+      structural_variants: 32170052,
+      short_variants_with_phenotype_assertions: 13365325,
+      short_variants_with_publications: 429376,
       short_variants_frequency_studies: 8,
-      structural_variants_with_phenotype_assertions: 1921150
+      structural_variants_with_phenotype_assertions: 320417
```
Comment (azangru) [inline src/ensembl/src/content/app/species/sample-data.ts:102]: this looks like quite a jump. Is this number correct?

File: src/ensembl/src/content/app/species/sample-data.ts [hunk 2]
```diff
@@ -40,66 +40,66 @@ export const sampleData: RawSpeciesStats = {
...
+      transcripts_per_gene: 1.22,
+      total_exons: 44532,
+      average_exon_length: 353.34,
+      average_exons_per_transcript: 2.39,
+      total_introns: 25915,
+      average_intron_length: 4372.89
     },
     homology_stats: {
-      coverage: 98.8
+      coverage: 86
```
Comment (azangru) [inline src/ensembl/src/content/app/species/sample-data.ts:94]: Surprised the homology stats a) reduced and b) by over 10%. Is this number correct?

File: src/ensembl/src/content/app/species/sample-data.ts [hunk 3]
```diff
@@ -316,7 +316,7 @@ export const sampleData: RawSpeciesStats = {
       average_intron_length: 203.57
     },
     homology_stats: {
-      coverage: 51.8
+      coverage: 87.3
```
Comment (azangru) [inline src/ensembl/src/content/app/species/sample-data.ts:319]: Also a strangely huge difference. Is this number correct?

**Review by ens-st3 — APPROVED** [review]

**Comment by ens-st3** [comment]
David is happy with the changes (due to changes in the number of genes, not the number of homologs)

File: src/ensembl/src/content/app/species/sample-data.ts [hunk 4]
```diff
@@ -31,75 +31,75 @@ export const sampleData: RawSpeciesStats = {
...
     variation_stats: {
-      short_variants: 713730678,
-      structural_variants: 32132047,
-      short_variants_with_phenotype_assertions: 13336864,
-      short_variants_with_publications: 303288,
-      short_variants_frequency_studies: 8,
-      structural_variants_with_phenotype_assertions: 1921150
+      structural_variants: 32170052,
+      short_variant_frequency_studies: 8,
+      structural_variants_with_phenotype_assertions: 320417,
```
Comment (ens-st3) [inline src/ensembl/src/content/app/species/sample-data.ts]: I am waiting to hear on this

Comment (ens-st3) [reply src/ensembl/src/content/app/species/sample-data.ts]: It's intentional, we removed a whole bunch of phenotypes of 'not reported' previously imported from dbVar

File: src/ensembl/src/content/app/species/sample-data.ts [hunk 5]
```diff
@@ -112,7 +112,7 @@ export const sampleData: RawSpeciesStats = {
       total_genome_length: 3234834689,
       total_coding_sequence_length: 33914472,
       total_gap_length: 243146473,
-      spanned_gaps: 242,
+      spaned_gaps: 242,
```
[the remark this answers is not in the document]
Comment (msmanoj) [reply src/ensembl/src/content/app/species/sample-data.ts]: @ens-st3  this is from an older commit. 
The key hasn't been changed in the latest commit: https://github.com/Ensembl/ensembl-client/pull/546/files#diff-90e73514e5e364d89d884afb58fb5dcb39b8f4dd27be4305bddd7b7fe7534164L328

**Review by azangru — APPROVED** [review]

**Pull request merged**